### DISCUSSION TITLE
Certificate Renewal + new key

### DISCUSSION THREAD

**Discussion by DConcord** [smallstep/certificates#664] [Q&A] [answered]
When using the "step ca renew", I've noticed that the same private key is re-used. Is there any way to specify that a new key is generated with each renewal?

**Comment by tashian** [comment, marked as answer]
Hi @DConcord, `step-ca`'s API supports this, and there's [an open PR](https://github.com/smallstep/cli/pull/514) for this that @tommy-56 just created a few days ago, should be reviewed & merged into the client very soon.

**Reply by DConcord** [reply]
Perfect, thanks!
